assets/js/main.js

```diff
@@ -3,6 +3,36 @@ $(function(){
     $('.sidenav, .overlay').toggleClass('is-visible');
     e.preventDefault();
   });
+
+  var generateCodeSnippets = function(content, previewBox) {
+
+    var self = this;
+
+    this.parseCode = function(previewBox) {
```

Comment:
Declaring a function within generateCodeSnippets would contain it as well, so I'm still confused it's being declared on this rather the just being declared. Maybe to make unit testing easier? Maybe just add a TODO comment to rethink the structure of it.